state/src/font_style.rs

```diff
@@ -31,6 +36,12 @@ impl FontStyle {
             ..FontStyle::default()
         }
     }
+
+    pub fn to_skia_font_style(&self) -> skia_safe::font_style::FontStyle {
```

Comment:
Maybe this could be made by using the `From` trait?

```rust
impl From<FontStyle> for skia_safe::font_style::FontStyle {
    fn from(value: FontStyle) -> Self {
        skia_safe::font_style::FontStyle::new(value.font_weight, value.font_width, value.font_slant)
    }
}
```

Reply:
Should be done, let me know if anything I changed is wrong.